src/components/Swap.tsx

```diff
@@ -66,13 +66,26 @@ export const Swap = (): JSX.Element => {
 	const [fromValue, setFromValue] = useState('');
 	const [toValue, setToValue] = useState('');
 	const { data: tokensData } = useTokens();
-
 	const [fromToken, setFromToken] = useState<Token | undefined>();
 	const [fromAmount, setFromAmount] = useState<string>();
 	const [toToken, setToToken] = useState<Token>();
 	const [toAmount, setToAmount] = useState<string>();
 	const { data: tokenBalances } = useTokenBalances();
 
+	const conversionRates = useMemo(() => {
```

Comment:
do we really need `useMemo` here?

Good reading https://kentcdodds.com/blog/usememo-and-usecallback/.